lib/src/markdown.service.ts

```diff
@@ -423,4 +442,12 @@ export class MarkdownService {
           : line;
       }).join('\n');
   }
+
+  private useExtensions() {
+    if (this.extensions) {
+      for (const ext of this.extensions) {
```

Comment:
the `marked.use` function can take multiple parameters that are extensions.

`marked.use(extension1, extension2)`

You can remove this for loop and use the spread operator
```js
marked.use(...this.extensions);
```
is the same as
```js
for (const ext of this.extensions) {
  marked.use(ext);
}
```

Reply:
done and inlined it inside the `parseMarked` as you have suggested in the other discussion